src/components/pages/race/charts/county-chart.module.scss

```diff
@@ -1,14 +1,27 @@
 @import '~scss/colors.module.scss';
 @import '~scss/breakpoints.module.scss';
 
+.hispanic-latino {
+  fill: $color-palette-100;
+}
+.nhpi-alone {
+  fill: $color-palette-200;
+}
 .black-african-american-alone {
-  fill: #ffad4a;
+  fill: $color-palette-300;
 }
 .white-alone {
-  fill: #585bc1;
+  fill: $color-palette-400;
 }
-.hispanic-latino {
-  fill: green;
+.aian-alone {
+  fill: $color-palette-500;
+}
+.asian-alone {
+  fill: $color-palette-600;
+}
+
+.two-or-more {
+  fill: red;
 }
 
```

Comment:
Based on the legend SASS file, should `.two-or-more` have `fill: #444;`?

Reply:
Yes! I noticed that and changed both to `#555`

 [(054ce20)](https://github.com/COVID19Tracking/website/pull/993/commits/054ce2016a0cb3bac2689d36918b381aade5be65)